src/mongocxx/bulk_write.cpp

```diff
@@ -84,6 +84,10 @@ void bulk_write::append(const model::write& operation) {
     }
 }
 
+bsoncxx::stdx::optional<void*> bulk_write::implementation() const {
```

Comment:
FYI, if you include mongocxx/stdx.h the bsoncxx polyfills are automatically imported into the mongocxx namespace.